acceptance/helpers.rb

```diff
@@ -87,8 +87,9 @@ def require_master_collection(args)
       if args.is_a?(Symbol)
         collection = args.to_s
 
-        # puppet_collection_for doesn't know about nightly collections
-        unless collection.include?(server_collection)
+        # Dirty, temporary workaround for when we have puppet8 agent nightlies but not server nightlies
+        # Once we have puppet8 server releases, remove the second conditional
+        unless collection.include?(server_collection) || collection.include?('nightly')
```

Comment:
Should this check for `puppet8-nightly` specifically or is it a general issue for puppet7-nightly too?

Reply:
Good catch, I've updated the commit and limited it to `puppet8-nightly`